### DISCUSSION TITLE
pion to pion, using browser implementation

### DISCUSSION THREAD

**Discussion by unicomp21** [pion/webrtc#2001] [Q&A] [answered]
Assuming use of a websocket server for signaling, should this example work for browser-to-browser as well?  With the wasm implementation?

https://github.com/pion/webrtc/blob/master/examples/pion-to-pion/offer/main.go

**Comment by unicomp21** [comment, marked as answer]
In the example, why is there no need for a createAnswer call?

**Reply by ashellunts** [reply]
https://github.com/pion/webrtc/blob/master/examples/pion-to-pion/answer/main.go#L105
